### ISSUE TITLE
Is this item system get bugged on Arc Warden's Tempest Double

### ISSUE THREAD

**Issue by rosesagaming** [ValveSoftware/Dota2-Gameplay#14903]
### Item name

Hand of Midas

### Description

I haven't tried in Turbo or any other game mode, but it looks like this Hand of Midas system is not right on Tempest Double! The video is on my X account, so feel free to check [here.](https://twitter.com/RosesaGaming/status/1745040091184316462)

### Example Match ID and Timestamp

00:00 - 01:43

**Comment by Aluerie** [comment]
you lost 3.3k value while doing so as intended.

**Comment by rosesagaming** [comment]
Yeah, I believe that bug is on the tempest double and not on the real one.

**Comment by Mikaols** [comment]
The problem in question appears to be exclusive to the <a href="https://basketrandom.pro/">basket random</a>  and not the original.
